sdk/iot/provisioning/src/az_iot_provisioning_client.c

```diff
@@ -11,13 +12,19 @@
 static const az_span provisioning_service_api_version
     = AZ_SPAN_LITERAL_FROM_STR("/api-version=" AZ_IOT_PROVISIONING_SERVICE_VERSION);
 
-static const az_span str_dps = AZ_SPAN_LITERAL_FROM_STR("$dps");
-static const az_span str_registrations = AZ_SPAN_LITERAL_FROM_STR("/registrations/");
+static const az_span user_agent_version_prefix = AZ_SPAN_LITERAL_FROM_STR("&ClientVersion=");
+
+static const az_span str_dps_registrations_res
+    = AZ_SPAN_LITERAL_FROM_STR("$dps/registrations/res/");
+static const int32_t idx_registrations_start = 4;
```

Comment:
Please add a comment explaining where these numbers came from. Something like right after `$dps` in `str_dps_registrations_res`.